src/app/public/components/archive-search/archive-search.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+// import { ComponentFixture, TestBed } from '@angular/core/testing';
+
+// import { ArchiveSearchComponent } from './archive-search.component';
+
+// describe('ArchiveSearchComponent', () => {
+//   let component: ArchiveSearchComponent;
+//   let fixture: ComponentFixture<ArchiveSearchComponent>;
+
+//   beforeEach(async () => {
+//     await TestBed.configureTestingModule({
+//       declarations: [ ArchiveSearchComponent ]
+//     })
+//     .compileComponents();
+//   });
+
+//   beforeEach(() => {
+//     fixture = TestBed.createComponent(ArchiveSearchComponent);
+//     component = fixture.componentInstance;
+//     fixture.detectChanges();
+//   });
+
+//   it('should create', () => {
+//     expect(component).toBeTruthy();
+//   });
+// });
```

Comment:
@cecilia-donnelly I have deleted the lines because all the tests were failling because of a typescript error with unknown. This test was fixed on the implement api search branch. Is that ok?

Reply:
@crisnicandrei works for me since these branches will all be merged together. Thanks!